ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install_dcgm.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Any difference for dcgm as compared to regular Ubuntu-HPC image? If not, should we use a common script?

Reply:
dcgm is identified to have similar structure for Ubuntu. Created a common script.